src/code/audio_synthesis.c

```diff
@@ -858,33 +858,35 @@ Acmd* AudioSynth_ProcessNote(s32 noteIndex, NoteSubEu* noteSubEu, NoteSynthesisS
 
                 switch (sample->codec) {
                     case CODEC_ADPCM:
-                        frameSize = 9;
-                        skipInitialSamples = 16;
+                        frameSize = 9; // 16 samples (32 bytes) compressed into 8 bytes + 1 header byte
```

Comment:
these comments are nice, I feel like they could even go on the `SampleCodec` `CODEC_` enum?

Reply:
(I didn't think you would duplicate them but why not idk)